app/src/main/java/com/example/habitsmasher/MainActivity.java

```diff
@@ -25,30 +28,30 @@
  */
 public class MainActivity extends AppCompatActivity {
 
-    private ActivityMainBinding binding;
-    AppBarConfiguration appBarConfiguration;
+    private ActivityMainBinding _binding;
+    AppBarConfiguration _appBarConfiguration;
+    private NavController _navController;
```

Comment:
Just changing these global variables to start with a `_` to match the style in our project.